images/access-setup/Dockerfile

```diff
@@ -4,15 +4,16 @@ RUN mkdir /workspace && chmod 777 /workspace && chown 65532:65532 /workspace
 ENV HOME /tmp/home
 RUN mkdir $HOME && chmod 777 $HOME && chown 65532:65532 $HOME
 
-RUN microdnf install gzip tar
+RUN microdnf install -y gzip-1.9 tar-1.30 && microdnf clean all
 RUN JQ_VERSION=1.6 && \
     curl -sSL -o /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-$JQ_VERSION/jq-linux64 && \
     chmod 755 /usr/local/bin/jq
 RUN KUBE_VERSION=v1.24.0 && \
     curl -L -o /usr/local/bin/kubectl "https://dl.k8s.io/release/$KUBE_VERSION/bin/linux/amd64/kubectl" && \
     chmod 755 /usr/local/bin/kubectl
-RUN KCP_VERSION=v0.7.4 && \
-    curl -L -o /tmp/kubectl-kcp.tgz "https://github.com/kcp-dev/kcp/releases/download/$KCP_VERSION/kubectl-kcp-plugin_${KCP_VERSION:1}_linux_amd64.tar.gz" && \
+SHELL ["/bin/sh", "-eo", "pipefail", "-c"]
+RUN KCP_VERSION=0.7.4 && \
+    curl -L -o /tmp/kubectl-kcp.tgz "https://github.com/kcp-dev/kcp/releases/download/v$KCP_VERSION/kubectl-kcp-plugin_${KCP_VERSION}_linux_amd64.tar.gz" && \
```

Comment:
hadolint complained about `${KCP_VERSION:1}`?

Reply:
Yes, hadolint complained a waring  `SC3060 warning: In POSIX sh, string replacement is undefined.`. To keep the url of kcp plugin simple, I modified `KCP_VERSION` from `v0.7.4` to `0.7.4`

Reply:
As far as I have seen there is not such a thing as kcp version 0.7.4. kcp version name is v0.7.4. This is not following semver but it is the way it is today.

Reply:
sure, I will following semver to fix it, thanks

Reply:
fixed